### PR TITLE
Added basic outline to wiki page

### PR DESCRIPTION
**Pull request by sgosline**
I don't know enough about wikis to add more.

### CODE REVIEW

File: articles/wikis.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+{% tab Python %} {% highlight python %}
+projWiki = Wiki(title='Data Summary', owner = myProj ) markdown = '''* Cell growth look normally distributed
+There is evidence of inverse growth between these two cell lines ''' projWiki['markdown'] = markdown projWiki = syn.store(projWiki) 
+{% endhighlight %} {% endtab %}
+
+{% tab R %} {% highlight r %} library(synapseClient); 
+placeholderText <- "* Cell growth look normally distributed\n* There is evidence of inverse growth between these two cell lines." wiki <- WikiPage(owner=myProject, title="Analysis summary", markdown=placeholderText) wiki <- synStore(wiki) 
+{%endhighlight %} {% endtab %}
+
+{% tab Web %} Select **Tools** then select **Edit Project/Folder/File wiki** {% endtab %}
```
Comment (kdaily) [inline articles/wikis.md]: Wiki capitalized

File: articles/wikis.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  
+  The easiest way [add how to, I have never figured it out]
+  
+  * Query on Synapse table
+  
+If you have an existing Synapse table related to this project you can query selected columns using this Synapse table widget.  To do so select 'Edit Project Wiki' from the 'Tools' directory and from the 'Insert' directory select 'Query on Synapse Table'. Once the widget opens you can type the query according the the [Synapse table query language](ISTHEREALINKFORTHIS?). 
+
+  * Query on files/folders
+
+You can also query specific files and folderst.  To do so select 'Edit Project Wiki' from the 'Tools' directory and from the 'Insert' directory select 'Query on Files/Folders'. Once the widget opens you can type the query according the the [Synapse query language]().
```
Comment (kdaily) [inline articles/wikis.md]: typo (folders)

File: articles/wikis.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  * Team Badge
+
+###Wiki Versioning
+We should describe how to revert back to older versions of the wiki, if possible.  [I do not know enougha bout this!]
+
+###Deleting a wiki
+It is possible to remove a wiki from a File or Folder. There are two ways to do this:
+1. Select *Tools* and select *Delete Wiki Page*
+2. Select *Tools* and select *Edit wiki* and then click the red 'Delete page' button.
+
```
Comment (kdaily) [inline articles/wikis.md]: Note this is unrecoverable and all versions are deleted.

**Comment by sgosline** [comment]
I updated the issues, but cannot re-assign.

**Pull request merged**